README.md

```diff
@@ -6,104 +6,17 @@ tools and environments that are required to execute pipeline steps.
-
-### Docker Files
-
-The following Dockerfiles will not be published as images. You have to build them on your own.
-
-| Name | Description | Link |
-|------|-------------|------|
-| Node-rfc | Interact with CTS+ having NW ABAP < 7.50 | [/node-rfc](https://github.com/SAP/devops-docker-images/tree/master/node-rfc)|
-| xs-cli | XS command-line client | [/xs-cli](https://github.com/SAP/devops-docker-images/tree/master/xs-cli)|
-
-## General Requirements
-
-A [Docker](https://www.docker.com/) environment is needed to build and run Docker images. You should be familiar with basic Docker commands to build and run these images. In case you need to fetch the Dockerfiles and this project's sources to build them locally, a [Git client](https://git-scm.com/) is required.
-
-Each individual Dockerfile may have additional requirements. Those requirements are documented with each Dockerfile.
-
-## Download and Installation
-
-To download and install Docker please follow the instructions at the [Docker website](https://www.docker.com/get-started) according your operating system.
-
-You can consume these images in three different flavors:
-
-1. Build locally and run
-
-    Clone this repository, change directories to the desired Dockerfile and build it:
-    
-    ````
-    git clone https://github.com/SAP/devops-docker-images
-    cd devops-docker-images/<specific-image>
-    docker build .
-    docker run ...
-    ````
-
-    Specific instructions how to run the containers are stored within the same directory.
-
-2. Pull from hub.docker.com
-
-    We build the Dockerfiles for your convenience and store them on https://hub.docker.com/.
-    
-    ````
-    docker pull <image-name>:<version>
-    docker run ...
-    ````
-
-3. Via project "Piper"
-
-    In case you are using [project "Piper"](https://sap.github.io/jenkins-library/) you can configure certain steps 
-    to use docker images instead of the local Jenkins environment. These steps will automatically pull and run these 
-    images.
- 
-### Setting up Jenkins Server
-The `cx-server` is a toolkit that is developed to manage the lifecycle of the Jenkins server.
-In order to use the toolkit, you need a file named `cx-server` and a configuration file `server.cfg`. 
-You can generate these files using the docker command
-
-```sh
-docker run -it --rm -u $(id -u):$(id -g) -v "${PWD}":/cx-server/mount/ ppiper/cx-server-companion:latest init-cx-server
-``` 
-
-Once the files are generated in the current directory, you can launch the below command to start the Jenkins server.
-
-```sh
-./cx-server start
-```
-
-If you would like to customize the Jenkins, [the operations guide](https://github.com/SAP/devops-docker-images/blob/master/docs/operations/cx-server-operations-guide.md) will provide more information on this along with the lifecycle management of the Jenkins. 
+**Note:** This repository has been split up.
```

Comment:
Warum behalten wir die obige Tabelle nicht?

Reply:
Weil sie auf nicht mehr existente Dockerfiles verweist und sich dann doppelt mit der Liste der neuen Repos. Ja, man kann auch die Links auf die neuen Repos in die Tabelle machen, aber vielleicht wird das nicht deutlich genug dass die nicht mehr in diesem Repo sind. Ich werde das bei Gelegenheit nochmal überarbeiten.